### PR TITLE
gha: add docker 27.1.0

### PR DESCRIPTION
**Pull request by thaJeztah**
**What I did**

**Related issue**

**(not mandatory) A picture of a cute animal, if possible in relation to what you did**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -145,6 +145,7 @@ jobs:
           - 25.0.5
           - 26.1.4
           - 27.0.3
+          - 27.1.0
```
Comment (glours) [inline .github/workflows/ci.yml:148]: Can we just keep the last version for each major release to limit resources and CI time consumption?

Comment (thaJeztah) [reply .github/workflows/ci.yml:148]: Yup, makes sense! Let me do so. Also considering to swap 24.x for 23.x (as 24 is no longer maintained, but 23 is maintained by Mirantis for their LTS release); will do that separately though (maybe there was a reason we didn't do 23.0)

Comment (thaJeztah) [reply .github/workflows/ci.yml:147]: updated; PTAL 🫶

**Review by glours — APPROVED** [review]
LGTM

**Pull request merged**